test/customer_test.exs

```diff
@@ -1,5 +1,6 @@
 defmodule Stripe.CustomerTest do
   use ExUnit.Case, async: true
+  import Stripe.Fixture.Token
```

Comment:
There is some ambiguity so we will have to manually assign the alias value.

```
alias Stripe.Fixture.Token, as: TokenFixture
```
something like that.